torchsynth/module.py

```diff
@@ -193,6 +205,21 @@ def p(self, parameter_id: str) -> T:
         assert value.shape == (self.batch_size,)
         return value
 
+    def to(self, device=None, **kwargs):
+        """
+        Overriding the to call for nn.Module to transfer this module to device. This
+        makes sure that the ParameterRanges for each ModuleParamter and the globals
+        are also transferred to the correct device.
+        """
+        self.update_device(device)
+        return super().to(device=device, **kwargs)
+
+    def update_device(self, device=None):
+        self.synthglobals.to(device)
```

Comment:
Make a comment here explaining why we move parameter ranges to device but not the parameters themselves?